manual/graph-ts/chain/tendermint/proto/src/t.v1.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package t.v1;
```

Comment:
Where does the name `t` come from?

Reply:
This is to not have namespace conflict in graph-ts. Those proto file are the source for as-proto to generate graph-ts encode/decode binding